gG3TT3Rr/HW1/main1.py

```diff
@@ -0,0 +1,13 @@
+print("What is your name?")
+name = input()
```

Comment:
better without the print method and see for this:
name = input("What is your name?")